charts/templates/configmap-probe.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: monitor-liveness-probe
```

Comment:
The name should be changed to monitor-liveness-probe-{{ .Release.Name }} to avoid conflict when having multiple groups.